hadoop-common-project/hadoop-common/src/main/native/native.vcxproj.filters

```diff
@@ -48,19 +46,16 @@
     <ClCompile Include="src\org\apache\hadoop\util\NativeCodeLoader.c">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="src\org\apache\hadoop\io\compress\lz4\lz4.c">
-      <Filter>Source Files</Filter>
-    </ClCompile>
-    <ClCompile Include="src\org\apache\hadoop\io\compress\lz4\lz4hc.c">
+    <ClCompile Include="src\org\apache\hadoop\security\JniBasedUnixGroupsMappingWin.c">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="src\org\apache\hadoop\io\compress\lz4\Lz4Compressor.c">
+    <ClCompile Include="src\org\apache\hadoop\io\compress\zlib\ZlibCompressor.c">
```

Comment:
Why are we changing this?

Reply:
These changes were automatically made by Visual Studio when I ran the retarget command.

The `lz4` library was removed in https://github.com/apache/hadoop/pull/2576. The author didn't remove them from Visual Studio, so I removed them - https://github.com/apache/hadoop/commit/44910b53f383f8ab092b7f9889fbf353e057d3f7. But I forgot to remove them from `native.vcxproj.filters` file (the current file). When I retargeted the native project, Visual Studio was smart enough to detect that these files don't exist and has removed them for us.